site.conf.in

```diff
@@ -1,3 +1,3 @@
 INHERIT += "own-mirrors"
-SOURCE_MIRROR_URL ?= "%CACHEURL%/downloads/"
-SSTATE_MIRRORS ?= "file://.* %CACHEURL%/sstate-cache/PATH"
+SOURCE_MIRROR_URL ?= "file://%CACHEURL%/downloads/"
```

Comment:
Could you move it into the env.YOCTO_CACHE_URL? So this could then use both a NFS and HTTP cache.

Reply:
I suppose I could, but it would become problematic when trying to copy artifacts to the CACHE_URL (shells doesn't handle "file://")

Reply:
But I suppose we could do a check to see if the CACHEURL begins with http:// or file:// and act accordingly, if we want to easily switch between the two. On the other hand, publishing to the http:// mirror has to be handled separately in that case anyway, which it isn't in the NFS case.